index.html

```diff
@@ -189,18 +189,21 @@ <h2 id="project_tagline">A simple glossary for Ember terms and concepts</h2>
 <h3>
 <a id="observer" class="anchor" href="#observer" aria-hidden="true"><span class="octicon octicon-link"></span></a>Observer</h3>
 
-<p>Triggers a function when a primitive property changes.</p>
+<p>Immediately triggers a side-effecting function, when a primitive property changes.</p>
 
 <div class="highlight highlight-js"><pre>Ember.<span class="pl-c1">Object</span>.extend({
-  valueObserver<span class="pl-k">:</span> Ember.observer(<span class="pl-s"><span class="pl-pds">'</span>value<span class="pl-pds">'</span></span>, <span class="pl-k">function</span>() {
-    <span class="pl-c">// Executes whenever the "value" property changes</span>
+  onIsFormValidChanged<span class="pl-k">:</span> Ember.observer(<span class="pl-s"><span class="pl-pds">'</span>isFormValid<span class="pl-pds">'</span></span>, <span class="pl-k">function</span>() {
+    <span class="pl-c">// Executes whenever the "isFormValid" property changes</span>
+    if (!this.get('isFormValid')) {
```

Comment:
I don't know how to syntax highlight this code snippet, like you've done elsewhere in this file. This index.html almost looks compiled. But I couldn't find the source in any branch. Can you help highlight this?